### ISSUE TITLE
None of the descendent elements with "menuitem" role is tabbable  Violation UI component must have at least one tabbable descendant for keyboard access

### ISSUE THREAD

**Issue by emassino** [carbon-design-system/carbon#13085]
### Package

carbon-components-react

### Browser

Chrome

### Package version

6.116.3

### React version

16.9

### Automated testing tool and ruleset

IBM Accessibility Equal Access Accessibility Checker

### Description

I get a violation running the scan on my application but it can also reproduced on the storybook

### WCAG 2.1 Violation

2.1.1 Keyboard

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-overflowmenu--default

### Steps to reproduce

Run the scan on the overflow menu opened. I get the error also on the official storybook. Element location:
<ul
    style="top: 82px; position: absolute; opacity: 1; left: 42px; right: auto;"
    data-floating-menu-direction="bottom"
    aria-label="overflow-menu"
    role="menu"
    tabindex="-1"
    class="cds--overflow-menu-options cds--overflow-menu-options--open cds--overflow-menu-options--md">

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by adarsh-ibm** [comment]
hi, am also facing the same issue in one of our application. Any update on this?

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/12728
